### DISCUSSION TITLE
Possible Feature Request: Add Number of Corners Data to Circuits

### DISCUSSION THREAD

**Discussion by Hayessc2** [f1db/f1db#35] [Q&A] [answered]
Firstly, I want to share my appreciation for f1db - it's a fantastic resource!

I'm wondering about the number of corners for each circuit. Is there currently a way to identify this data within f1db, or would I need to add this manually?

If it's not available yet, I'd love to request this information as a feature.

Thanks!

**Comment by marceloverdijk** [comment]
Hi @Hayessc2,

First of all thanks for your nice words!

Currently I don't have that information (the number of corners for each circuit), but we could add that indeed, same as we have the course length.
Do you know if the information is somewhere available?

**Reply by marceloverdijk** [reply, marked as answer]
I've created https://github.com/f1db/f1db/issues/54 to include this information to the database.

**Reply by Hayessc2** [reply]
Thank you!!
